styles.css

```diff
@@ -1 +1,77 @@
-/* this will be the styles file */
+* {
+  background-color: black;
+}
+body {
+  margin: 0px 160px;
+}
+.logo {
+  display: flex;
+  color: #fff;
+  justify-self: start;
+  align-items: center;
+}
+nav {
+  color: #fff;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  gap: 48px;
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 26px;
```

Comment:
do we need a bottom padding as well??

Reply:
yes sorry for that i just forgot  now its fixed